apf/apfArrayData.h

```diff
@@ -4,19 +4,32 @@
  * This work is open source software, licensed under the terms of the
  * BSD license as described in the LICENSE file in the top-level directory.
  */
-
 #ifndef APFARRAYDATA_H
 #define APFARRAYDATA_H
-
 #include "apfFieldData.h"
-
 namespace apf {
+  template <class T>
+  void freezeFieldData(FieldBase* base);
+  template <class T>
+  void unfreezeFieldData(FieldBase* base);
+  template <class T>
+  class ArrayDataOf : public FieldDataOf<T> {
```

Comment:
Why does this class need to be exposed? Is it used somewhere else in the commit?

Reply:
My mistake. I was using this in a previous attempt to get this working.